text/0000-discord.md

```diff
@@ -0,0 +1,170 @@
+- Start Date: 07/11/2018
+- RFC PR: (leave this empty)
+- Ember Issue: (leave this empty)
+
+# RFC to move the Ember community chat to Discord
+
+## Summary
+
+Encourage the Ember community to adopt Discord for real-time chat.
+
+## Motivation
+
+Real-time chat is essential to the function of online communities, particularly in open source. Chat fosters an informal, interactive style of communication that is important for building relationships, sharing community norms, coordinating on projects, and brainstorming ideas.
+
+The Ember community predominantly uses [a Slack instance](https://ember-community-slackin.herokuapp.com/) as the gathering place of choice. While we have benefited enormously from Slack, there are significant downsides as well.
+
+### Loss of History
+
+Because we use Slack's free plan, the entire instance is limited to 10,000 messages in history at any time. Because of this hard cap, the amount of time messages persist continues to shrink as the community grows.
+
+It's hard to quantify exactly how painful this limitation is, as it means that new community members can't search for the answer to a question that was likely answered in the past. We can never go back to reference how or when a decision was made, which can mean decision-making feels less transparent that it should be.
+
+This limit applies not just to chat messages, but direct messages between community members as well. This leads to annoyance, as people have to ask for the same information over again if they forgot to save it, or data loss, as useful things like code snippets vanish into the ether.
+
+### Performance
+
+The architecture of the Slack native application relies on running a separate web application per Slack instance the user is signed into. For users who need to be in multiple Slack instances, this can add up to a significant tax on computer resources, particularly as the application starts up.
```

Comment:
I personally don't think this is a good point to make up a decision. Slack has become very frequently used on OSS projects. For instance I only use 1 slack instance, which has multiple workspaces for different OSS projects. If now one OSS projects decides to no longer use slack, this puts a burden on everyone who is contributing to multiple OSS projects, as they now need to use multiple clients.

Reply:
I think there needs to be clarification of the word “instance”. Slack workspaces or orgs are implemented as completely separate processes meaning the more slack orgs you belong to the less performant it becomes.

Anecdotally I find that discord takes much fewer resources than slack while in the background and about the same resources as one workspace when in the foreground.

Reply:
I personally think this section should be removed or reworded, it specifically states that performance isn't different enough to be a factor in the decision **but** at the same time the sections presence (in an RFC motivating a move) clearly indicates that it is...

Reply:
I am in like 10 discord servers and never have performance issues, some of them quite big. But with slack it just takes 1 tab to start creeping up resource usage. More than 1 slack and it is noticeable and why I can't idle in places I want to. I refuse to use so many resources for a damn chat application.